<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriggerEdit</class>
 <widget class="QDialog" name="TriggerEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>927</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Trigger</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="reactions_pattern" native="true">
     <layout class="QHBoxLayout" name="_3">
      <item>
       <widget class="QLabel" name="Pattern_label">
        <property name="text">
         <string>&amp;Trigger:</string>
        </property>
        <property name="buddy">
         <cstring>Pattern</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Pattern"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Text_label">
     <property name="text">
      <string>&amp;Send:</string>
     </property>
     <property name="buddy">
      <cstring>Text</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="Text"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_main">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="_options">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QFormLayout" name="_send">
         <item row="0" column="0">
          <widget class="QLabel" name="SendTo_label">
           <property name="text">
            <string>Send to:</string>
           </property>
           <property name="buddy">
            <cstring>SendTo</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="SendTo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>World</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Command</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Output</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Status</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Notepad (new)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Notepad (append)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Log file</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Notepad (replace)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>World (speedwalk delay)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Variable</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Execute</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Speedwalk</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Script</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>World (immediate)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Script (after omit)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Sequence_label">
           <property name="text">
            <string>Seq&amp;uence:</string>
           </property>
           <property name="buddy">
            <cstring>Sequence</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="Sequence">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>-30000</number>
           </property>
           <property name="maximum">
            <number>30000</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="Script_label">
           <property name="text">
            <string>Script:</string>
           </property>
           <property name="buddy">
            <cstring>Script</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="Script"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="Group_label">
           <property name="text">
            <string>Group:</string>
           </property>
           <property name="buddy">
            <cstring>Group</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="Group"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="Variable_label">
           <property name="visible">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Variable:</string>
           </property>
           <property name="buddy">
            <cstring>Variable</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="Variable">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="Label"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="Label_label">
           <property name="text">
            <string>&amp;Label:</string>
           </property>
           <property name="buddy">
            <cstring>Label</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="triggers_sound">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Sound</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="_soundfile">
            <item>
             <widget class="QLineEdit" name="Sound"/>
            </item>
            <item>
             <widget class="QToolButton" name="Sound_browse">
              <property name="text">
               <string>Browse...</string>
              </property>
              <property name="icon">
               <iconset theme="document-open"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="Sound_none">
              <property name="text">
               <string>No sound</string>
              </property>
              <property name="icon">
               <iconset theme="user-trash-symbolic"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="Sound_test">
              <property name="text">
               <string>Test</string>
              </property>
              <property name="icon">
               <iconset theme="audio-volume-high-symbolic"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="SoundIfInactive">
            <property name="text">
             <string>Only if inactive</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="triggers_style">
         <property name="title">
          <string>Change colour and style</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="3">
           <widget class="ColorPickerButton" name="ForegroundColor">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="baseSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="ChangeBackground">
            <property name="text">
             <string>Set background colour:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="ColorPickerButton" name="BackgroundColor">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="baseSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="MakeItalic">
            <property name="text">
             <string>Italic</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="MakeUnderline">
            <property name="text">
             <string>Underline</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="ChangeForeground">
            <property name="text">
             <string>Set foreground colour:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="MakeBold">
            <property name="text">
             <string>Bold</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_checkboxes">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="Enabled">
         <property name="text">
          <string>En&amp;abled</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="OmitFromOutput">
         <property name="text">
          <string>O&amp;mit from output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="OmitFromLog">
         <property name="text">
          <string>&amp;Omit from log file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="OneShot">
         <property name="text">
          <string>One-shot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Temporary">
         <property name="text">
          <string>Temporary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="IgnoreCase">
         <property name="text">
          <string>&amp;Ignore case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="KeepEvaluating">
         <property name="text">
          <string>&amp;Keep evaluating</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ExpandVariables">
         <property name="text">
          <string>Expand variables</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="LowercaseWildcard">
         <property name="text">
          <string>Make wildcards lower-case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="IsRegex">
         <property name="text">
          <string>Regular expression</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Repeats">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Repeat on same line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="MultiLine">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Multi-line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="triggers_multi_line" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignmentFlag::AlignRight">
           <widget class="QLabel" name="LinesToMatch_label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Lines:</string>
            </property>
            <property name="buddy">
             <cstring>LinesToMatch</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="LinesToMatch">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerButton</class>
   <extends>QToolButton</extends>
   <header>colorpickerbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TriggerEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>917</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TriggerEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>917</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IsRegex</sender>
   <signal>toggled(bool)</signal>
   <receiver>MultiLine</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>614</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>828</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IsRegex</sender>
   <signal>toggled(bool)</signal>
   <receiver>triggers_multi_line</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>614</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>873</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MultiLine</sender>
   <signal>toggled(bool)</signal>
   <receiver>LinesToMatch</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>828</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>864</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MultiLine</sender>
   <signal>toggled(bool)</signal>
   <receiver>LinesToMatch_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>828</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>864</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChangeForeground</sender>
   <signal>toggled(bool)</signal>
   <receiver>ForegroundColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>695</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>697</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChangeBackground</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackgroundColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>731</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>733</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Sound_none</sender>
   <signal>clicked()</signal>
   <receiver>Sound</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>589</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
